site/layouts/partials/stylesheet.html

```diff
@@ -1,4 +1,5 @@
-{{- "<!-- Bootstrap core CSS -->" | safeHTML }}
+<link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/@docsearch/css@3">
```

Comment:
maybe add a check here?

```suggestion
{{ if eq .Page.Layout "docs" -}}
<link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/@docsearch/css@3">
{{- end }}
```